### ISSUE TITLE
SA1122 (UseStringEmptyForEmptyStrings) is reported for default parameters

### ISSUE THREAD

**Issue by pdelvo** [DotNetAnalyzers/StyleCopAnalyzers#444]
Currently SA1122 UseStringEmptyForEmptyStrings is reported for default parameters:
```csharp
public class Foo
{
    public Foo(string bar = "")
    {
        // ...
    }
}
```

Default values have to be constant so string.Empty cant be used here.

**Comment by sharwell** [comment]
I'm working on this.
